### PR TITLE
Fix up the import from code-dot-org

### PR DESCRIPTION
**Pull request by Hamms**
Update import/export to require/module.exports, update some require paths, and clean up some other no-longer-supported syntax

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@code-dot-org/maze",
-  "version": "0.0.1",
+  "version": "1.0.0",
```
Comment (balderdash) [inline package.json]: :rocket:

File: test/unit/mazeMap.test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const MazeMap = require('../../src/mazeMap');
+const Cell = require('../../src/cell');
+
+describe("MazeMap", function () {
```
Comment (joshlory) [inline test/unit/mazeMap.test.js]: For these tests, where is `describe` defined?

Can we make it possible to run this test as simply as `node mazeMap.test.js`?

File: package.json [hunk 3]
```diff
@@ -41,6 +35,10 @@
     "file-loader": "^1.1.6",
     "html-webpack-plugin": "^2.30.1",
     "jest": "^22.1.3",
+    "prop-types": "^15.6.0",
+    "react": "~15.4.0",
+    "react-dom": "~15.4.0",
+    "redux": "^3.7.2",
```
Comment (joshlory) [inline package.json]: Would be cool if the eventual API for integrating the `@code-dot-org/maze` module didn't depend on React+Redux, but out of scope for this PR.

File: src/utils.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+ */
+module.exports.isCollectorSkin = function isCollectorSkin(skinId) {
+  return (/collector(_night)?/).test(skinId);
+}
+
+/**
+ * Is skin scrat
+ */
+module.exports.isScratSkin = function isScratSkin(skinId) {
+  return (/scrat/).test(skinId);
```
Comment (joshlory) [inline src/utils.js]: Tiny style nit: parens around the regex aren't needed — `/scrat/.test(skinId)` is fine.

File: test/unit/mazeMap.test.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    const serializedValues = [
+      [{tileType: 1, value: 4}, {tileType: 2, value: 3}],
+      [{tileType: 3, value: 2}, {tileType: 4, value: 1}]
+    ];
+    const mazeMap = MazeMap.deserialize(serializedValues, Cell);
+
+    expect(mazeMap.getCell(0, 0)).toEqual(new Cell(1, 4));
+    expect(mazeMap.getCell(0, 1)).toEqual(new Cell(2, 3));
+    expect(mazeMap.getCell(1, 0)).toEqual(new Cell(3, 2));
+    expect(mazeMap.getCell(1, 1)).toEqual(new Cell(4, 1));
```
Comment (joshlory) [inline test/unit/mazeMap.test.js]: Same question for `expect` — where is it defined / which assertion library does this use?

File: package.json (hunk 3 above)
Comment (Hamms) [reply package.json]: oh, actually it totally can already

File: test/unit/mazeMap.test.js (hunk 2 above)
Comment (Hamms) [reply test/unit/mazeMap.test.js]: `describe`, `it`, and `expect` are all provided by jest (and by the "jest" env in eslintrc for linting purposes). They can all be explicitly included, I was just gonna do that in a separate pass

**Pull request merged**